zanata-war/src/main/java/org/zanata/webtrans/client/resources/WebTransMessages.java

```diff
@@ -381,8 +381,11 @@ String undoUnsuccessful(@PluralCount int unsuccessfulCount,
     @DefaultMessage("Page size")
     String pageSize();
 
-    @DefaultMessage("Read only")
-    String readOnly();
+    @DefaultMessage("This project-version is readonly. It cannot be edited.")
+    String readOnlyTooltip();
+
+    @DefaultMessage("This project-version is obsolete. It cannot be edited.")
```

Comment:
Yes. It is `obsolete` in all the code, so I had that in mind. I'll change the string to "archived"